include/re_net.h

```diff
@@ -40,7 +40,8 @@ struct sa;
 
 /* Net generic */
 int  net_hostaddr(int af, struct sa *ip);
-int  net_dst_source_addr_get(struct sa *dst, struct sa *ip);
+int  net_dst_source_addr_get(const struct sa *dst, struct sa *ip);
```

Comment:
can you please make this change into a separate PR ?

is it just changing the type from non-const to const ?

Reply:
Now only making parameter `dst` to `const`. PR: https://github.com/baresip/re/pull/109